x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_field.scss

```
@@ -0,0 +1,19 @@
+.appSearchResultField {
+  font-size: $euiSize * .875; // 14px
```

We should use an [EUI font size variable](https://github.com/elastic/eui/blob/master/src/global_styling/variables/_typography.scss#L40) here:

```suggestion
  font-size: $euiFontSizeS;
```

---

Oh that's rad, I didn't realize that existed.